terms-and-conditions-page/content.en.json

```diff
@@ -0,0 +1,80 @@
+{
+  "title": "Terms and conditions",
+  "termsAndConditions": {
+    "termsOfuse": "By using this service you’re agreeing to the following terms of use.",
+    "govPrivacy": "These include the GOV.UK <a href='https://www.gov.uk/help/privacy-policy'>privacy policy</a> and <a href='https://www.gov.uk/help/terms-conditions'>terms and conditions</a>.",
+    "renderedGovPrivacy": "These include the GOV.UK privacy policy and terms and conditions.",
+    "links": {
+      "privacy": {
+        "name": "privacy policy"
+      },
+      "terms": {
+        "name": "terms and conditions"
+      }
+    }
+  },
+  "relatedLinks": {
+    "title": "Related links",
+    "links": {
+      "cookie": {
+        "text": "Cookie policy",
+        "url": "/cookie-policy"
+      },
+      "privacy": {
+        "text": "GOV.UK privacy policy",
+        "url":"https://www.gov.uk/help/privacy-policy"
```

Comment:
It's not a good idea to add URLs into a content file, where that file will be translated into another language. Ideally we should create a urls.js file defining every external URL we link to. This makes it far easier to maintain and update. TYA currently has one: https://git.reform.hmcts.net/sscs/track-your-appeal-frontend/blob/master/app/urls.js

Reply:
I'll create a separate ticket for this as it appears the landing pages have the same problem, we can address them all together.

Reply:
ticket is -> https://tools.hmcts.net/jira/browse/SSCS-2370